### PR TITLE
fix(fa.ts): update `fa` translation

### PR DESCRIPTION
**Pull request by mahsarajabpour**

### CODE REVIEW

File: packages/vuetify/src/locale/fa.ts [hunk 1]
```diff
@@ -69,22 +69,22 @@ export default {
...
+      last: 'صفحه‌ی آخر',
     },
   },
   rating: {
     ariaLabel: {
       item: 'Rating {0} of {1}',
     },
   },
-  loading: 'Loading...',
+  loading: 'بارگذاری...',
```
Comment (arminmon) [inline packages/vuetify/src/locale/fa.ts:85]: A suggestion for the sake of accuracy.
```suggestion
  loading: 'در حال بارگذاری...'
```

File: packages/vuetify/src/locale/fa.ts [hunk 2]
```diff
@@ -69,22 +69,22 @@ export default {
...
   infiniteScroll: {
-    loadMore: 'Load more',
-    empty: 'No more',
+    loadMore: 'بارگذاری بیشتر',
+    empty: 'چیزی وجود ندارد',
```
[the remark this answers is not in the document]
Comment (smortexa) [reply packages/vuetify/src/locale/fa.ts:88]: To be more concise and fit more with the context:
```suggestion
    empty: 'پایان',
```

File: packages/vuetify/src/locale/fa.ts [hunk 3]
```diff
@@ -31,15 +31,15 @@ export default {
...
     range: {
-      title: 'Select dates',
-      header: 'Enter dates',
+      title: 'تاریخ‌ها را انتخاب کنید',
+      header: 'تاریخ‌ها را وارد کنید',
     },
-    title: 'Select date',
-    header: 'Enter date',
+    title: 'تاریخ‌ را انتخاب کنید',
+    header: 'تاریخ را وارد کنید',
```
Comment (nasermirzaei89) [inline packages/vuetify/src/locale/fa.ts:40]: I suggest the "Enter date" translation to be shorter, as the font is too big by default
image

**Comment by johnleider** [comment]
Please implement feedback

**Comment by johnleider** [comment]
Last call on this. Any changes remaining?

**Review by johnleider — APPROVED** [review]

**Pull request merged**